update_ign.py

```diff
@@ -0,0 +1,224 @@
+#The script does the following things:
+#Update boostrap.ign with hostname.
+#According to the number of the master count, create the JSON files, and add hostname/network-scripts.
+#According to the number of the worker count, create the JSON files, and add hostname/network-scripts.
+
+#Assumptions
+#1 Interface Names: The RHCOS-4.6 OS Machines comes up with defined interface names viz. ens2, ens4, ens5
+#2 Machine Names: The machine hostnames are based on <Infra-Id>-<bootstrap/master/worker>-<Number>
+#3 Ignition Files: The new ignition files generated as per node viz. <Infra-Id>-bootstrap/master/worker-<index>-ignition.json 
+#4 config.yaml: The script requires user-defined input file - config.yaml to be configured proper
```

Comment:
It's not clear that you are referring to the config.yaml in the same repo. Suggest saying -

Configure the config.yaml with the relevant information.